### ISSUE TITLE
prepare for a new 4.6.1 release

### ISSUE THREAD

**Issue by sezero** [libxmp/libxmp#768]
Can we make a new 4.6.1 release by the end of this November?

Open tickets in 4.6.1 milestone: https://github.com/libxmp/libxmp/milestones/4.6.1

CC: @AliceLR
CC: @cmatsuoka

**Comment by AliceLR** [comment]
Yes, November works for me. I am still caught up in MegaZeux nonsense (regressions from last release, SDL3's impending release, and my VM images are horribly out of date), but I should be able to focus on the remaining libxmp 4.6.1 issues soon.

**Comment by AliceLR** [comment]
Quick refresher on the pending tasks, which I'm fairly certain I can get done tomorrow while it's still technically 2024 :)

* #765: The details of this are built into the corresponding OpenMPT test so this will be quick.
* #784: This requires at most a couple of lines in each read event implementation.
* #786: I'm just going to add "%MODPATH%/Instruments" as an extra song file search path.
* There is a new mixer crash most likely related to the ADPCM sample loader fix; probably just need to round up the reduced length to a multiple of the frame size.

**Comment by sezero** [comment]
> edit: forgot to add the Pattern Loop changelog entry for libxmp-lite, can follow up on that unless that patch needs changes.

Added a one-liner for that in #812

**Comment by sezero** [comment]
Is there anything else left ?

**Comment by sezero** [comment]
Ah, there is the new mixer crash you mentioned above.

I'll wait for your fix and final green light for release.

**Comment by AliceLR** [comment]
Yes, I am trying to finish up the fix for that. I was wrong; it's not ADPCM, but rather a rare mixer crash related to the sample reverse effect and low playback sample rates.

**Comment by sezero** [comment]
#819 is in. Waiting for your final word.

**Comment by AliceLR** [comment]
Getting a lot of MSan use-of-uninitialized-value in the mixer doing a corpus merge, probably not anything serious but I will look into it after I sleep. (Sorry for yet another delay...)

**Comment by sezero** [comment]
> Getting a lot of MSan use-of-uninitialized-value in the mixer doing a corpus merge, probably not anything serious but I will look into it after I sleep.

I ran under valgrind with a few it and s3m mods and it was silent. But my mods possibly won't cover everything.

> (Sorry for yet another delay...)

No, better be late than being sorry later. We don't want to make a release with issues present.

**Comment by AliceLR** [comment]
Fix incoming shortly, I just want to let the corpus merge complete this time to make sure I actually got it. This time it actually *was* the extra bytes that were non-multiple of the sample frame size causing problems: for stereo samples they wouldn't get initialized.

**Comment by AliceLR** [comment]
I think 4.6.1 should be good to go if you're OK with it.

**Comment by sezero** [comment]
OK. Should we include #821 or should it wait?

**Comment by AliceLR** [comment]
> OK. Should we include #821 or should it wait?

Sorry, missed that—just merged it.

**Comment by sezero** [comment]
OK. I'll directly push (w/o a PR) to set release date in ChangeLogs, and then make the release shortly.

**Comment by sezero** [comment]
OK, released both at sf.net and here at github.
